src/ORM/Marshaller.php

```diff
@@ -133,7 +138,7 @@ public function one(array $data, array $options = [])
             $columnType = $schema->columnType($key);
             if (isset($propertyMap[$key])) {
                 $assoc = $propertyMap[$key]['association'];
-                $value = $this->_marshalAssociation($assoc, $value, $propertyMap[$key]);
+                $value = $this->_marshalAssociation($assoc, $value, $propertyMap[$key] + $marshallOptions);
```

Comment:
not imho. If you specify a primaryKey int that assoc and it does not exist the new row will be created, so is not necesary to force anything. Maybe I skipped something or someone has other ideas.

If so what would you recomend? to check if it is a belongToMany assoc before add the extra options?